tendrl/notifier/objects/definition/notifier.yaml

```diff
@@ -48,6 +48,9 @@ namespace.alerting:
         source:
           help: 'The process raising the alert'
           type: String
+        delivery:
```

Comment:
yep I just meant create a bug and link it here so we know what needs to be done...
But anyway you could track it using https://github.com/Tendrl/notifier/issues/29
So, that resolves the comment :)